python/fastsim/demos/demo.py

```diff
@@ -482,92 +482,94 @@ def get_sim_drive_vec(
 # Includes example of how to load cycle from dict
 
 # %%
-veh = fsim.vehicle.Vehicle.from_vehdb(1)  # load vehicle model
-output = {}
-
-results_df = pd.DataFrame()
-t_start = time.perf_counter()
-for trp in list(drive_cycs_df.nrel_trip_id.unique()):
-    pnts = drive_cycs_df[drive_cycs_df['nrel_trip_id'] == trp].copy()
-    pnts['time_local'] = pd.to_datetime(pnts['timestamp'])
-
-    cyc = {}
-    cyc['cycGrade'] = np.zeros(len(pnts))
-    cyc['mps'] = np.array(
-        pnts['speed_mph'] / fsim.params.MPH_PER_MPS)  # MPH to MPS conversion
-    cyc['time_s'] = np.array(
-        np.cumsum(
-            (pnts['time_local'] -
-             pnts['time_local'].shift()).fillna(pd.Timedelta(seconds=0)).astype('timedelta64[s]')
+with fsim.utils.suppress_logging():
```

Comment:
Can we change instances of `fsim.utils.suppress_logging():` to a single `fsim.utils.disable_logging()` at the top of the file?

Reply:
@kylecarow, I prefer the latter because I'd like it to be really obvious that we're suppressing logging so that we remember to fix the cause of the logging at some point in the future.  Does this seem ok?

Reply:
I'm fine with either way really. Do you mean keeping it as is?